pkg/controllers/attractor/stateless-lb-frontend.go

```diff
@@ -202,7 +202,7 @@ func (l *LoadBalancer) insertParameters(dep *appsv1.Deployment) *appsv1.Deployme
 	}
 	ret.Spec.Template.Spec.InitContainers[0].Args = []string{
 		"-c",
-		common.GetLoadBalancerSysCtl(l.trench),
+		fmt.Sprintf("%s ; %s", common.GetLoadBalancerSysCtl(l.trench), common.GetFrontendSysCtl()),
```

Comment:
Could be added here otherwise: https://github.com/Nordix/Meridio/blob/v1.0.9/pkg/controllers/common/models.go#L42

Reply:
Yeah, but I wanted to handle it separately, even though currently LB and FE share the same POD, the settings are FE specific.
There were plans to support a non-composite setups in the future, in which case I figured separating these could help sort things out.